### PR TITLE
feat: add titles to charts / flamegraphs

### PR DESCRIPTION
**Pull request by dogfrogfog**
#1204

### CODE REVIEW

**Comment by Rperry2174** [comment]
I know its still draft, but two things I'd add in:
image
Theres some extra padding for `diff-instructions-wrapper` that makes the gap larger. Lets remove that

also let's do font size 18 instead of 22 for the titles

**Comment by eh-am** [comment]
https://user-images.githubusercontent.com/6951209/177356441-70fee6b5-e827-41e1-afd6-882eeb7cecb3.mp4

Also the content shift is bit annoying, can we always have something in the header (fixed height, or a dummy element dunno)?

File: webapp/javascript/pages/ContinuousComparisonView.tsx [hunk 1]
```diff
@@ -102,11 +97,8 @@ function ComparisonApp() {
               right: { from: rightFrom, to: rightUntil, color: rightColor },
             }}
             timezone={timezone}
-            title={
-              <TimelineTitle
-                titleKey={isSidesHasSameUnits ? leftSide?.metadata.units : ''}
-              />
-            }
+            // using leftSide because comparison view (for now) should match the application at the top of the screen
```
Comment (dogfrogfog) [inline webapp/javascript/pages/ContinuousComparisonView.tsx:100]: shouldn't we expand(or remove) this comment ?

**Comment by dogfrogfog** [comment]
latest version demo

https://user-images.githubusercontent.com/47758224/177386299-9f1f53b1-899b-48a5-b9a1-4896e2ec6545.mov

**Review by eh-am — APPROVED** [review]
Looks good! Don't forget to add to Standalone export too (https://github.com/pyroscope-io/pyroscope/blob/main/webapp/javascript/standalone.tsx)!

**Pull request merged**